ephios/static/ephios/css/main.css

```diff
@@ -83,3 +83,7 @@ This hack fixes that. Inspired by https://github.com/select2/select2/issues/4220
 .select2 {
     width: 100% !important;
 }
+
+.no-display {
+    display: none;
+}
```

Comment:
Why not use `d-none` from bootstrap?

Reply:
because the boostrap one uses !important and the jquery slideDown does not override the !important flag, so the content is not shown